rotkehlchen/chain/ethereum/decoding/constants.py

```diff
@@ -38,3 +38,6 @@
 
     string_to_ethereum_address('0x340d693ED55d7bA167D184ea76Ea2Fd092a35BDc'): 'Uphold.com',
 }
+
+GAS_COUNTERPARTY = 'gas'
+XDAI_COUNTERPARTY = 'gnosis-chain'
```

Comment:
Well since you change the name you should also change the variable name

```suggestion
CPT_GNOSIS_CHAIN = 'gnosis-chain'
```